### PR TITLE
removed docs, changed Examples to match FMI.jl

### PR DESCRIPTION
**Pull request by 0815Creeper**
does not contain a jupyter-notebook yet. can be merged, if you want to keep this seperate, otherwise we will need the working export-example of the BouncingBall.jl FMU first

### CODE REVIEW

File: .github/workflows/Example.yml [hunk 1]
```diff
@@ -1,76 +1,103 @@
...
+  jupyter:
     runs-on: ${{ matrix.os }}
     strategy:
-      fail-fast: true
+      fail-fast: false
       matrix:
-        julia-version: ['1.9']
+        os: [windows-latest] # ubuntu-latest
+        file-name: []
+        julia-version: ['1.10'] # 1.9
```
Comment (ThummeTo) [inline .github/workflows/Example.yml:23]: better `lts`

Comment (0815Creeper) [reply .github/workflows/Example.yml:23]: impossible as discussed some time ago with FMI.jl: there is no jupyter notebook kernal named "julia-lts". but there is one named "julia-1.10". See line 46 for the occurance

**Comment by 0815Creeper** [comment]
ready to merge. 
be aware, that .fmu files are gitignored at the present time.
Example action trys to trigger FMI-Doc building, for that to succede secrets.FMI_DOC_TRIGGER_PAT has to be provided (see [here](https://github.com/peter-evans/repository-dispatch?tab=readme-ov-file#token) for more info on that

File: examples/jupyter-src/Export-BouncingBall.ipynb [hunk 2]
```diff
@@ -0,0 +1,377 @@
+{
+ "cells": [
```
Comment (ThummeTo) [inline examples/jupyter-src/Export-BouncingBall.ipynb:2]: Please add at the very top (highly visible) that this is just a placeholder example and will be replaced by something better soon :-)

Comment (0815Creeper) [reply examples/jupyter-src/Export-BouncingBall.ipynb:2]: done

**Pull request merged**